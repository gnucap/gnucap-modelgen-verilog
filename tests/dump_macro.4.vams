
`define thing

`ifdef thing
module reached1
endmodule
`elsif FOO
notreached0
`ifdef thing
notreached1
`elsif FOO
notreached2
`else
notreached3
`endif
notreached4
`else
parse_error2
`endif
