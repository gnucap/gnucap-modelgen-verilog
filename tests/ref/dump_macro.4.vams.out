

module reached1();
endmodule



